make_ocr_features/utils/OCR_preprocessing.py

```diff
@@ -0,0 +1,128 @@
+# Author: Anis ZAKARI <anis.zakari@outlook.fr>
+
+import re
+import multiprocess as mp
+import numpy as np
+
+###make df###
+
+def get_row_from_json_for_df(json_line: dict) -> list:
+    """ extract items from json and returns a row to put in a dataframe 
+    Parameters
+    ----------
+    json_line: dict 
+        dictionary loaded from a line in the jsonl file.
+    
+    Returns
+    -------
+    row: list
+        list with items [code, ocr_texts, keys]
+            code: barcode of the product
+            ocr_texts: concatenated texts coming from all the images
+            keys: keys of all the images; for example 004/150/000/7229/2.json  <-- "2" is the key.
+    """
+    code = json_line['code']
+    if "ocrs" in json_line:
+        texts = []
+        keys =  list(json_line['ocrs'].keys())
+        for key in keys:
+            ocr_text = json_line['ocrs'][key]['text']
+            texts.append(ocr_text)
+    else:
+        texts = []
+    row = [code, " ".join(texts), keys]
+    return row
+
+def append_item_ocr_text_from_json_to_dict(json_line: dict, ocr_text_dict: dict) -> dict:
+    """ extracts ocr text from all images, for a given barcode and 
+    appends it to the ocr dictionary. 
+    
+    Parameters
+    ----------
+    json_line: dict 
+        dictionary loaded from a line in the jsonl file.
+    ocr_text_dict: dict
+        dictionary in the following format {barcode: {key: ocr_text}}
+    """
+    
+    code = str(json_line['code'])
+    if "ocrs" in json_line:
+        keys =  json_line['ocrs'].keys()
+        if len(keys) >0:
+            ocr_text_dict[code] = {}
+            for key in keys:
+                ocr_text_dict[code][key] = json_line['ocrs'][key]['text']
+
+
+### cleaner ###
+
+def text_cleaner(text:str) -> str:
+    """ takes a text as input and cleans it"""
+
+    dont_take = ["kj", "kcal", "kj", "total", "free", "net", "ingredients", "ingredient", "et", "de", "fat", "mg", "cg", "g", "kg", "ml", "cl", "l", "kl", "per", "pour", "valeur", "or", "le", "la", "dont", "consommer", "poids", "net", "www", "com", "which", "of", "wt"]
```

Comment:
`dont_take` is also defined in OCR_word_selection.py. Can it be refactored into a single definition?